src/components/About/About.js

```diff
@@ -1,10 +1,13 @@
 import React, { Component } from 'react';
 import '../../styles/About.css';
+import OurStory from "./OurStory.js";
+
 
 class About extends Component {
   render() {
     return (
-      <div className="About" id="About">
+      <div className="About">
```

Comment:
The `id="About"` is needed to enable the anchor link to the About page. Let's do <div className="About" id="About">.